src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ADT_A01/ADT01-23-expected.json

```diff
@@ -91,10 +91,10 @@
         "status": "unknown",
         "location": [
           {
-            "status": "active",
             "location": {
-              "reference": "Location/5f49ca69-cbda-3b30-8480-fa750dfee4a2"
-            }
+              "reference": "Location/"
```

Comment:
This output is incorrect. It is caused by the `Encounter` template, which uses a Handlebars style to generate ID: `"reference":"Location/{{generateUUID PV1-3}}",`. Could you please help to create a working item about it?

Reply:
Sure, I'll create a task to fix this issue.

Reply:
Thanks!